ctapipe/conftest.py

```diff
@@ -62,6 +62,42 @@ def test_my_thing(test_event):
     return deepcopy(_global_example_event)
 
 
+@pytest.fixture(scope="function")
```

Comment:
I think the scope should be 'session' here, since with function scope this is re-run for every test (which is a bit slow). This only needs to be loaded once (assuming no test modifies it). 

Alternately, if you want to be careful to avoid modification by tests, you can keep it "function", but  make it a generator and add at the end:
```py3
while True:
     yield deepcopy(event)
```
(I think that works at least) see https://docs.pytest.org/en/stable/fixture.html#dynamic-scope

Reply:
With the `while True` I get

> fixture function has more than one 'yield':

Reply:
I think generators are only used to be able to run clean up code after a fixture isn't needed anymore